communication/socket/tcp/send/wsaioctl_transmitpackets_callbackfunc.yml

```
@@ -0,0 +1,28 @@
+rule:
+  meta:
+    name: obtains TransmitPackets callback function via WSAIoctl  
+    description: The TransmitPackets function transmits in-memory data or file data over a connected socket. The TransmitPackets function uses the operating system cache manager to retrieve file data, locking memory for the minimum time required to transmit and resulting in efficient, high-performance transmission.
```

for long line, use YAML folded style:
```suggestion
    description: >-
      The TransmitPackets function transmits in-memory data or file data over a connected socket. 
      The TransmitPackets function uses the operating system cache manager to 
      retrieve file data, locking memory for the minimum time required to transmit and 
      resulting in efficient, high-performance transmission.
```

---

https://stackoverflow.com/a/3790497/87207